cmd/analyzer/analyzer.go

```diff
@@ -190,9 +191,9 @@ func provisionStreamHealthcore(ctx context.Context, cli cliFlags) *health.Core {
 	return healthcore
 }
 
-func provisionDataAnalytics(cli cliFlags) (*views.Client, *usage.Client) {
+func provisionDataAnalytics(cli cliFlags) (*views.Client, *usage.Client, *ai.Client) {
 	if cli.disableBigQuery {
-		return nil, nil
+		return nil, nil, nil
```

Comment:
I guess we could still have an ai client without bigquery support 🤔 

Unless we'll be adding other bigquery stuff later then let's just leave it like this

Reply:
Oh yeah I've just moved the `ai.NewClient` call to the `Run` func